### DISCUSSION TITLE
Why isn't this project respecting the MIT license of morphdom?

### DISCUSSION THREAD

**Discussion by bcardarella** [livewire/livewire#2642] [4. General]
The original morphdom is MIT which has a license that requires any derivative work to reference the original: https://github.com/patrick-steele-idem/morphdom/blob/master/LICENSE#L12-L13

Is there any reason why this isn't being done for the fork included in this project?

**Comment by calebporzio** [comment]
Hi there. Great point. Oversight on my part. Would you mind letting me know what is needed or ideally submitting a PR to give proper attribution? Thanks!
